libs/directx/directx.cpp

```diff
@@ -132,11 +144,13 @@ HL_PRIM void HL_NAME(clear_color)( dx_pointer *rt, double r, double g, double b,
 	color[1] = (float)g;
 	color[2] = (float)b;
 	color[3] = (float)a;
-	driver->context->ClearRenderTargetView((ID3D11RenderTargetView*)rt,color);
+	ID3D11RenderTargetView* renderTarget = (ID3D11RenderTargetView*)rt;
+	driver->context->ClearRenderTargetView(renderTarget,color);
+	driver->context->OMSetRenderTargets(1, &renderTarget, depthBuffer);

```

Comment:
This is a violation of the API rules : this should be handled at the driver level !